### DISCUSSION TITLE
Date

### DISCUSSION THREAD

**Discussion by Sbyrd3** [ChurchApps/ChurchAppsSupport#479] [Chums] [answered]
When I add a date for example 9/1/24 it saves it as 8/31/24... it does this on all dates...saves it as one day earlier.

**Comment by jzongker** [comment]
It's most likely a time zone issue where the dates are getting converted to GMT.  If you can let me know a specific place or two where this is happening, I should be able to fix it.  I tested with birthdate, anniversary, group sessions, giving batches, and plans and they all seem to be working ok.

**Comment by Sbyrd3** [comment]
Hey! It’s happening when under the “shepherding” tab - if I add first time they visited - for example I put 01/02/2020 and it changed it to 01/01/2020

**Comment by jzongker** [comment, marked as answer]
Thank you for the clarification.  It was indeed the time zone issue I suspected and has been fixed now.  Let me know if you still encounter any problems.
